src/pages/Metrics/components/MetricsSimilarList/MetricsSimilarList.scss

```diff
@@ -0,0 +1,57 @@
+.metrics-similar-photos {
```

Comment:
looks like a bottom margin is needed on this element on mobile, please check

Reply:
Looks like at the moment the element has intrinsic browser margins. I would recommend to reset them (probably mixin reset-list can be used?) and set them explicitly 
<img width="513" alt="Screenshot 2023-04-19 at 11 50 45 AM" src="https://user-images.githubusercontent.com/39587602/232990274-6afbd602-d303-4d43-91f7-66ad46791f9d.png">